sockeye/constants.py

```diff
@@ -195,7 +195,8 @@
 JSON_SUFFIX = ".json"
 VOCAB_SRC_PREFIX = "vocab.src"
 VOCAB_SRC_NAME = VOCAB_SRC_PREFIX + ".%d" + JSON_SUFFIX
-VOCAB_TRG_NAME = "vocab.trg" + JSON_SUFFIX
```

Comment:
As long as we're changing this, shouldn't it be ".tgt"?

Reply:
I am a big 'trg' believer :)